kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -1504,7 +1503,7 @@ private void makePatternWindow(EntityFacade patternFacade, ViewProperties viewPr
     }
 
     private void makeGenEditWindow(EntityFacade entityFacade, ViewProperties viewProperties) {
-        System.out.println("Launching General editing window: " + entityFacade);
+//        System.out.println("Launching General editing window: " + entityFacade);
```

Comment:
I forgot to remove. This was already there, but also, this can throw a runtime exception if the entityFacade has trouble `toString()`